### ISSUE TITLE
Rename and rewire ast.pow_spec config params: corr_freq in Hz, and fix the unwired defaults

### ISSUE THREAD

**Issue by chrisfinlay** [epfl-radio-astro/tabascal#117]
`ast.pow_spec` takes an inverse scale where `rfi` takes a physical one, and half its keys are unwired.

The astronomical and RFI GP priors describe the same kind of object — a power spectrum with a knee, a roll-off and a truncation — but their config sections disagree on units, on naming, and on which keys are read at all. Three separable problems, entangled in the same handful of lines.

## 1. `k0_freq` is an inverse scale in seconds; `rfi.corr_freq` is a bandwidth in Hz

`rfi` takes physical quantities and inverts them internally:

```yaml
rfi:
  corr_time: 24         # seconds
  corr_freq: 1e6        # Hz
```

`ast` takes the already-inverted quantity:

```yaml
ast:
  pow_spec:
    k0_freq: 1
```

The k grids are `fftfreq(n, dx)` with `dx = [chan_width, int_time]` (`fft_gp.py:162`), so the frequency-axis k has units of **1/Hz, i.e. seconds** — `k0_freq` is a knee expressed as a *delay*, and its reciprocal is a correlation bandwidth. The time axis is the other way round: k is in Hz and `k0_time` is a fringe rate.

`rfi` converts with `k0s = 1 / (2 * pi * [corr_freq, corr_time])` (`rfi_signal.py:683`), so the two spellings are exact reciprocals up to `2*pi` and either can be canonical.

**The current value is not merely awkwardly expressed — it is inert.** The delay axis runs to `1 / (2 * chan_width)`, which is 2.39 us for the 209 kHz channels of the shipped configs, independent of channel count. The configured `k0_freq: 1` is a knee at **one second**, 4.2e5 times beyond the largest delay the data can represent, so the frequency roll-off is flat everywhere it is evaluated:

| `n_freq` | delay samples | max delay | `k0_freq: 1` sits |
|---|---|---|---|
| 1 | 1 (the zero mode) | 0 | nowhere — the knee has no effect at all |
| 32 | 64 | 2.39 us | 4.2e5x beyond the axis |
| 1024 | 2048 | 2.39 us | 4.2e5x beyond the axis |

Every shipped config is single channel, which is why nobody has noticed (see also #109). Expressed as a delay, `1 s` is visibly wrong at a glance — delays are microseconds. Expressed as `k0_freq: 1` it looks like a perfectly ordinary setting. That is the interpretability argument in one line.

**Wanted:** the knee settable in units a user can sanity-check, with the conversion pinned by a test rather than inferred from the name.

## 2. `ast` needs no `corr_time`, and that should be said

The time-axis knee is not configured and should not be: `k0_time` is the maximum astronomical fringe rate from `max_ast_fringe_rate`, set by `fov_deg` and the baseline geometry (#91). Adding a `corr_time` to match `rfi` would be wrong — it is derived, not chosen.

**Wanted:** documented explicitly, so the asymmetry with `rfi` reads as deliberate rather than missing.

## 3. The base config's `ast.pow_spec` defaults are dead

`tab_config_base.yaml` ships:

```yaml
ast:
  pow_spec:
    P0: null
    gamma: 1.0
    k0: 1e-3
```

`GPVisAst.setup` reads (`ast_vis.py:40-44`):

```python
self.p0        = config.args["ast"]["pow_spec"]["p0"]
self.gammas    = config.args["ast"]["pow_spec"]["gammas"]
self.fov_deg   = config.args["ast"]["pow_spec"]["fov_deg"]
self.k0_freq   = config.args["ast"]["pow_spec"]["k0_freq"]
self.pk_cutoff = config.args["ast"]["pow_spec"]["cutoff"]
```

**Not one name matches.** `P0`/`gamma`/`k0` are read by nothing, and the five keys that *are* read have no defaults at all — omit any one from a user config and setup raises `KeyError` through `RuntimeError`. Every shipped config supplies all five, which is the only reason this is invisible; `examples/` and `ci/reframe/data/` even carry the dead names commented out above the live ones, so the rename happened once already and the base config never followed.

## 4. `gammas` and the cutoff are wired for `ast` and hard-coded for `rfi`

`ast` reads both from config. `rfi` reads neither — they are literals in two different components, with different values:

| | `gammas` | cutoff |
|---|---|---|
| `ComplexRFIVarAnt` (`rfi_signal.py:687`) | `[3, 3]` | `1e-9` |
| `ComplexRFIConstAnt` (`rfi_signal.py:972`) | `[1e2, 1e2]` | `1e-6` |
| `GPVisAst` | from config | from config |

`[3, 3]` and `[1e2, 1e2]` are very different priors — the latter is effectively an RBF. This half overlaps #111 (`rfi.pow_spec` is in the configs and read by nothing); the RFI side should be fixed there, and this issue should not duplicate it.

**Wanted:** the cutoff and roll-off exponent named consistently and read from config in every component that uses them.

## 5. `p0` is not in Jy, and nothing else pins the scale either

`rfi` normalises its power spectrum to a target variance (`rfi_signal.py:729`):

```python
scale_norm = self.gp_var / jnp.sum(self.pk)
self.pk = scale_norm * self.pk
```

so `sum(pk) == gp_var` and the synthesised signal has exactly the configured variance, whatever the roll-off does.

`ast` has no such step (`ast_vis.py:177`):

```python
sigma = lambda k0: jnp.sqrt(pow_spec_nd(self.ks, self.p0, [...], self.gammas) / self.pk.size)
```

The implied variance is `p0 * mean(shape)`, and `mean(shape)` depends on `gammas`, the knees, the cutoff and the grid. So `p0` is proportional to the prior width but with a constant nobody knows, and that constant moves whenever any other prior parameter is touched.

Measured on the shipped test configuration (`p0: 3e3`, `gammas: [5, 5]`, `cutoff: 1e-6`, `fov_deg: 5`, single channel, 120 times):

| change | surviving modes | implied prior std |
|---|---|---|
| shipped | 9 | **22.4 Jy** |
| `cutoff: 1e-3` | 5 | 30.0 Jy (1.34x) |
| `cutoff: 1e-9` | 11 | 20.2 Jy (0.90x) |
| `gammas: [3, 3]` | 15 | 17.6 Jy (0.79x) |

Neither `p0` (3000) nor `sqrt(p0)` (54.8) is the prior width. And tightening the truncation alone — an efficiency knob, not an amplitude — widens the prior by a third.

**Wanted:** the same normalisation `rfi` already uses, with the parameter exposed as a **standard deviation in Jy**, so that

- setting it to the visibility amplitude of an **uncontaminated channel** puts the true sky within **1 sigma** of the prior, and
- that calibration survives a change of `gammas`, `cutoff` or `fov_deg`, which it currently does not.

It has to be a standard deviation (Jy), not a variance (Jy^2), for "set it to the amplitude you see" to be literally true rather than approximately and only after squaring.

Worth documenting alongside it: how to pick the value, i.e. read the amplitude off a channel with no visible RFI, and what to do when no such channel exists.

### The normalisation target is not `rfi`'s, and the difference is a square

`rfi_A` is a per-antenna amplitude and the visibility is **quadratic** in it (`interferometry.py:445`):

```python
vis_rfi_fine = jnp.sum(rfi_A_[a1] * jnp.conjugate(rfi_A_[a2]) * jnp.exp(1.0j * (...)), axis=1)
```

So `rfi_A` carries units of sqrt(Jy) and `var(rfi_A)` is in Jy. That is why `rfi.var` defaulting to `max(|vis_obs|)` — a visibility amplitude in Jy — is dimensionally right rather than a muddle, and why `_read_estimate` takes `sqrt(|light_curves|)` to turn a measured light curve into an `rfi_A`.

`ast_vis` is not like that: `vis_ast` *is* the modelled quantity, **linear** in the latent. The two normalisations therefore differ by a square:

| | modelled quantity | relation to visibility | `sum(pk)` should equal |
|---|---|---|---|
| `rfi` | `rfi_A`, per antenna | `vis ~ rfi_A[a1] * conj(rfi_A[a2])` | visibility amplitude, Jy |
| `ast` | `vis_ast` | `vis = vis_ast` | (prior std in Jy)^2 |

Lifting `scale_norm = target / sum(pk)` across without squaring the target would set the ast prior width to `sqrt(amplitude)` — about 4.7 Jy for a 22 Jy sky, wrong by a factor of four and a half, and wrong in a direction that still looks like a plausible number.

It is also precisely because `ast` is linear that the guidance above works with no square root in it: set the parameter to the amplitude of an uncontaminated channel and the truth sits within 1 sigma. The same instruction for `rfi` would need squaring, which is a good reason not to give the two parameters the same name.

## 6. One shared spec, with the differences declared rather than duplicated

`ast` and `rfi` are the same object — a Fourier GP prior with a per-axis knee, a per-axis roll-off, a relative truncation and an overall amplitude. They differ in only three declarable ways. Building and validating the spec once, with the components declaring their differences, is better than keeping two hand-maintained parameter sets in step:

| | `rfi` | `ast` |
|---|---|---|
| amplitude convention | quadratic — `sum(pk)` is `var(rfi_A)`, equal to a visibility amplitude in Jy | linear — `sum(pk)` is `(prior std of vis_ast in Jy)^2` |
| time-axis knee | chosen, `corr_time` | derived from `fov_deg` and baseline geometry |
| freq-axis knee | chosen | chosen |

Everything else — the roll-off exponents, the truncation, the normalisation, the validation, the "exactly one spelling per axis" rule and the error messages — is common and should exist once.

### Each axis should accept any of its equivalent spellings

A knee is a single number in conjugate units; which spelling is natural depends on the field the user comes from. The canonical internal value stays the conjugate-domain `k0`, and the spec accepts whichever equivalent the user prefers, requiring exactly one:

| axis | conjugate units | spellings |
|---|---|---|
| frequency | seconds — a **delay** | correlation bandwidth (Hz), or **delay** (s / ns / us) |
| time | Hz — a **fringe rate** | correlation time (s), or fringe rate (Hz); for `ast`, derived from `fov_deg` |

**The delay spelling is not a convenience, it is the field's own convention.** Power spectra in 21cm work — HERA, HIRAX — are calculated and quoted on the delay axis, so a user coming from that literature already thinks in delays and already knows what a plausible one looks like. That is precisely the sanity check that would have caught `k0_freq: 1` immediately.

The same accept-either treatment applies to the amplitude: a user should be able to give a standard deviation or a variance and have the spec square or not, rather than having to remember which component wants which.

### Why one place

Every defect in this issue and in #111 is a consequence of the same structure being maintained twice: a dead `ast.pow_spec` block and a dead `rfi.pow_spec` block, `gammas` read on one side and hard-coded on the other, two RFI components disagreeing with each other, and a frequency knee whose units nobody had cause to question. A single spec with declared differences makes the asymmetries explicit — and reviewable — instead of implicit and drifting.

## Naming

Open, and worth settling before any of the above is implemented:

- **cutoff.** Config says `cutoff`, code says `pk_cutoff`, this issue's title says `pk_cut`. Pick one. It is a threshold *relative to* `p0` (modes with `pk < p0 * cutoff` are dropped), so whatever it is called should not read like an absolute power.
- **`gammas`.** Plural because it is per-axis `[freq, time]`, while the dead base-config key is singular `gamma`. If it is renamed, it should say what it is — a roll-off exponent — rather than a bare Greek letter.

At minimum keep the old names working as aliases, since existing user configs carry `k0_freq`, `gammas` and `cutoff`.

## Scope note

Any of items 1, 3 or 4 that changes a *value* rather than a name changes what the model infers. That means re-recording the pipeline references (`tests/test_tabascal_pipeline.py`, via `--record-refs`, across both precisions — see `docs/pipeline_tests.md`) and re-checking the CSCS reframe references, since the surviving k-mode count drives the parameter count. A pure rename with aliases changes neither and can land on its own; that is the easier PR to review and probably worth splitting out first.

## Acceptance

- [ ] One shared power-spectrum prior spec, built and validated in a single place, with the
      amplitude convention and the derived time knee declared per component
- [ ] Each axis accepts any of its equivalent spellings, exactly one at a time, with a clear
      error naming the alternatives; the frequency axis accepts a **delay** as well as a bandwidth
- [ ] Conversions between spellings pinned by tests, not inferred from names
- [ ] Base-config `ast.pow_spec` defaults match the keys the component actually reads, with a test that a config omitting the optional ones still builds
- [ ] Cutoff and roll-off exponent named consistently and read from config wherever they are used
- [ ] Old key names accepted as aliases, with a test
- [ ] Prior amplitude exposed as a standard deviation in Jy, normalised as `rfi` is, so that the
      configured value *is* the prior width and is invariant to `gammas` / `cutoff` / `fov_deg`
- [ ] A test pinning that invariance: same amplitude, different roll-off and truncation, same prior std
- [ ] Guidance documented on choosing it from an uncontaminated channel
- [ ] `fov_deg`-derived time knee documented as deliberately not configurable
- [ ] `docs/config.md` updated; references re-recorded if any default value moved

Related: #111 (`rfi.pow_spec` unwired), #115 (sweep `corr_time` for the perf config), #91 (`fov_deg` sets the time knee).

Part of #45 (unify GP config inputs into one prior spec).
